test_regress/t/t_vpi_dump_missing_scopes.cpp

```diff
@@ -0,0 +1,179 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
```

Comment:
There's got to be a way to just link t_vpi_dump.cpp in the .pl file right?

Reply:
Just add it to the verilog_flags2 => ['t_vpi_dump.cpp'].  That should work for all simulators.